common/beerocks/bwl/dummy/nl80211_client_dummy.h

```diff
@@ -44,6 +44,18 @@ class nl80211_client_dummy : public nl80211_client {
      */
     virtual bool get_sta_info(const std::string &local_interface_name,
                               const sMacAddr &sta_mac_address, sStaInfo &sta_info) override;
+
+    /**
+     * @brief Set the tx power limit object
+     *
+     * Set tx power limit for a radio
+     *
+     * @param[in] local_interface_name Virtual AP (VAP) interface name.
```

Comment:
I think it can be a physical radio interface as well as a virtual interface

Reply:
I think it will cause confusion, so lets keep it for radio only.